src/pages/404.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react'
+import { recordEvent } from '@/lib/analytics/recordEvent'
+import { Wrapper } from '@/templates/globals/wrapper'
+import { getGlobalElements } from '@/lib/drupal/getGlobalElements'
+import { CommonAndPopular } from '@/templates/common/commonAndPopular'
+
+const Error404Page = ({ headerFooterData }) => {
+  useEffect(() => {
+    recordEvent({ event: 'nav-404-error' })
+  })
+  return (
+    <Wrapper bannerData={[]} headerFooterData={headerFooterData}>
+      <div className="main maintenance-page" role="main">
+        <div className="primary">
+          <div className="row">
+            <div className="text-center usa-content">
+              <h3>Sorry — we can’t find that page</h3>
+              <p>Try the search box or one of the common questions below.</p>
+              <div className="feature va-flex va-flex--ctr">
+                <form
+                  acceptCharset="UTF-8"
+                  action="/search/"
+                  id="search_form"
+                  className="full-width"
+                  method="get"
+                >
+                  <div className="va-flex va-flex--top va-flex--jctr">
+                    <label htmlFor="mobile-query">Search:</label>
+                    <input
+                      autoComplete="off"
+                      className="usagov-search-autocomplete full-width"
+                      id="mobile-query"
+                      name="query"
+                      type="text"
+                    />
+                    <input type="submit" value="Search" />
+                  </div>
+                </form>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <CommonAndPopular />
+      </div>
+    </Wrapper>
+  )
+}
+
+export async function getStaticProps() {
```

Comment:
I think I would restructure this to match `[[...slug]].tsx` a little closer, unless there's a specific reason that the 404 page needs handled in this way.

`const { headerFooterData } = await getGlobalElements()` feels a bit cleaner than the let & try catch block. slug has the whole getStaticProps function in a try catch